jt-bandselect/src/main/java/it/geosolutions/jaiext/bandselect/BandSelectOpImage.java

```diff
@@ -47,6 +42,9 @@
 @SuppressWarnings("unchecked")
 public class BandSelectOpImage extends PointOpImage {
 
+    private final static boolean DATABUFFER_WORKAROUND =
```

Comment:
Does it make sense to put this variable somewhere more centralized, public and with a javadoc?

Reply:
I have initially put this into ImageUtilities and associated with a getter but at the end I was thinking it's a low level setting to be disabled when something goes wrong with the databuffer workarounds.